core/extension/gdextension_compat_hashes.cpp

```diff
@@ -450,6 +450,9 @@ void GDExtensionCompatHashes::initialize() {
 	mappings.insert("Line2D", {
 		{ "add_point", 468506575, 2654014372 },
 	});
+	mappings.insert("LightmapGIData", {
+		{ "add_user", 536974962, 862968125 },
+	});
```

Comment:
This shouldn't be added here. These mappings are for a very specific situation where we wanted to avoid using compatibility methods, however, it looks like this PR is adding a compatibility method.